### PR TITLE
[sram/dv] Minor DV doc update

### PR DESCRIPTION
**Pull request by weicaiyang**
1. added prim_prince, prim_lfsr are verified separately
2. added using different INSTR_EXEC parameter for ret and main sram

Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

File: hw/ip/sram_ctrl/doc/dv/index.md [hunk 1]
```diff
@@ -42,10 +42,9 @@ The following utilities provide generic helper tasks and functions to perform ac
 
 ### Compile-time configurations
 Two compile-time configurations are tested:
-* `sram_ctrl_main` - this configuration uses a 15-bit wide address space to
-  emulate the main SRAM used in the full design
-* `sram_ctrl_ret` - this configuration uses a 10-bit-wide address space to
-  emulate the retention SRAM used in the full design
+* `sram_ctrl_main` - this configuration uses a 15-bit wide address space to emulate the main SRAM used in the full design and enables parameter `INSTR_EXEC` to allow TLUL instruction fetching.
```
Comment (sriyerg) [inline hw/ip/sram_ctrl/doc/dv/index.md:45]:
```suggestion
* `sram_ctrl_main` - this configuration uses a 15-bit wide address space to emulate the main SRAM used in the full design and enables parameter `INSTR_EXEC` to allow TLUL instruction fetching when some runtime conditions are met.
```

File: hw/ip/sram_ctrl/doc/dv/index.md [hunk 2]
```diff
@@ -42,10 +42,9 @@ The following utilities provide generic helper tasks and functions to perform ac
...
+* `sram_ctrl_ret` - this configuration uses a 10-bit-wide address space to emulate the retention SRAM used in the full design and turns off parameter `INSTR_EXEC` to disable TLUL instruction fetching.
```
Comment (sriyerg) [inline hw/ip/sram_ctrl/doc/dv/index.md:46]:
```suggestion
* `sram_ctrl_ret` - this configuration uses a 10-bit-wide address space to emulate the retention SRAM used in the full design and turns off parameter `INSTR_EXEC` to disable TLUL instruction fetching altogether.
```

File: hw/ip/sram_ctrl/doc/dv/index.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ title: "SRAM_CTRL DV document"
 ## Goals
 * **DV**
   * Verify all SRAM_CTRL IP features by running dynamic simulations with a SV/UVM based testbench
-  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules
+  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules, except prim_prince and prim_lfsr, which are verified in separated testbenches.
```
Comment (sriyerg) [inline hw/ip/sram_ctrl/doc/dv/index.md:8]: Perhaps we should add a separate heading for this:
```
## Pre-verified sub-modules

The following sub-modules are pre-verified:
* [prim_prince](link-to-tb/reports)
* [prim_lfsr](...)

Only toggle coverage on the IOs of these sub-modules is enabled for coverage collection. 

```

**Review by sriyerg — APPROVED** [review]
LGTM, left some comments.

File: hw/ip/sram_ctrl/doc/dv/index.md (hunk 3 above)
Comment (weicaiyang) [reply hw/ip/sram_ctrl/doc/dv/index.md]: Good idea. Updated. Thanks

File: hw/ip/sram_ctrl/doc/dv/index.md (hunk 2 above)
Comment (weicaiyang) [reply hw/ip/sram_ctrl/doc/dv/index.md]: Updated.

File: hw/ip/sram_ctrl/doc/dv/index.md (hunk 1 above)
Comment (weicaiyang) [reply hw/ip/sram_ctrl/doc/dv/index.md]: Updated.

**Review by sriyerg — APPROVED** [review]

**Pull request merged**